src/Symfony/Component/PasswordHasher/PasswordHasherInterface.php

```diff
@@ -29,12 +29,12 @@ interface PasswordHasherInterface
      *
      * @throws InvalidPasswordException When the plain password is invalid, e.g. excessively long
      */
-    public function hash(string $plainPassword): string;
+    public function hash(#[\SensitiveParameter] string $plainPassword): string;
```

Comment:
The attribute has no effect on the interface, but is a remainder for implementers.